examples/README.md

```diff
@@ -2,10 +2,10 @@
 
 #### Usage Examples
 
-- [Simple examples](simple/README.md)
-- [ERC20](tokens/ERC20/): verifies OpenZeppelin, Solady, Solmate ERC20 tokens, and CurveTokenV3.
+- [Simple examples](simple/test/)
```

Comment:
we're not doing the README there anymore? we could also move the README at simple/test/README.md, so we would see the test files and the explainer

Reply:
the current readme is outdated, and needs to be updated to cover more examples there. we can update the link to the readme once it is updated.